docs/source/dev_release.md

```diff
@@ -59,8 +59,8 @@ Lerna will prompt you for version numbers for each of the changed npm packages i
 
 Go into the `jupyterlab_widgets` directory. Change `jupyterlab_widgets/_version.py` to reflect the new version number.
 ```
-python setup.py sdist bdist_wheel
-twine check dist/*
+pip install build
+python -m build
```

Comment:
Should we also update the packaging workflow to use these new commands?

https://github.com/jupyter-widgets/ipywidgets/blob/3108308d93c423845901500b108f45b4d7e9fc5b/.github/workflows/packaging.yml#L56

Reply:
Good point. Done!